.github/CODEOWNERS

```diff
@@ -1,106 +1,586 @@
+
+# ServiceLabel: %Network - Traffic Manager %Service Attention
+sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsuppgithub
+
+# ServiceLabel: %Network - Virtual WAN %Service Attention
+#/<NotInRepo>/            @vwansuppgithub
+
+# ServiceLabel: %Network - VPN Gateway %Service Attention
+#/<NotInRepo>/            @vpngwsuppgithub
+
+# ServiceLabel: %Network - Private Link %Service Attention
+#/<NotInRepo>/            @privlinksuppgithub
+
+# ServiceLabel: %Notification Hub %Service Attention
+/sdk/notificationhubs/Microsoft.Azure.Management.NotificationHubs/            @tjsomasundaram
+
+# PRLabel: %Operational Insights
+# ServiceLabel: %Operational Insights %Service Attention
+/sdk/operationalinsights/Microsoft.Azure.Ope*/  @divyajay @geneh @alongafni @AzmonLogA
+
+# ServiceLabel: %Policy %Service Attention
+#/<NotInRepo>/            @aperezcloud @kenieva
+
+# ServiceLabel: %Policy Insights %Service Attention
+/sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/           @kenieva
+
+# ServiceLabel: %PostgreSQL %Service Attention
+/sdk/postgresql/Microsoft.Azure.Management.PostgreSQL/            @sunilagarwal @lfittl-msft @sr-msft @niklarin
+
+# ServiceLabel: %Recovery Services Backup %Service Attention
+/sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/           @pvrk
+
+# ServiceLabel: %Recovery Services Site-Recovery %Service Attention
+/sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/            @Sharmistha-Rai
+
+# ServiceLabel: %Redis Cache %Service Attention
+/sdk/redis/Microsoft.Azure.Management.RedisCache/            @yegu-ms
+
+# ServiceLabel: %Relay %Service Attention
+/sdk/relay/Microsoft.Azure.Management.Relay/          @jfggdl
+
+# ServiceLabel: %Reservations %Service Attention
+/sdk/reservations/Microsoft.Azure.Management.Reservations/            @corquiri
+
+# ServiceLabel: %Resource Authorization %Service Attention
+#/<NotInRepo>/            @darshanhs90 @AshishGargMicrosoft
+
+# ServiceLabel: %Resource Graph %Service Attention
+/sdk/resourcegraph/Microsoft.Azure.Management.ResourceGraph/            @chiragg4u
+
+# ServiceLabel: %Resource Health %Service Attention
+#/<NotInRepo>/            @stephbaron
+
+# ServiceLabel: %Scheduler %Service Attention
+/sdk/scheduler/Microsoft.Azure.Management.Scheduler/            @derek1ee
+
+# ServiceLabel: %Security %Service Attention
+#/<NotInRepo>/            @chlahav
 
 # PRLabel: %Search
 /sdk/search/                                    @brjohnstmsft @arv100kri @bleroy @tg-msft @heaths
-/sdk/search/Microsoft.*/                        @brjohnstmsft @arv100kri @bleroy
 
-# PRLabel: %Service Bus
+# ServiceLabel: %Search %Service Attention
+/sdk/search/Microsoft.*/                        @brjohnstmsft @arv100kri @bleroy @tjacobhi @markheff @miwelsh
+
+# PRLabel: %Service Bus (Client)
 /sdk/servicebus/                                @JoshLove-msft @ShivangiReja @jsquire @MiYanni
```

Comment:
Does Shivangi still want to be spammed with SB stuff, or should we trim the list?

Reply:
I think we should remove her, but I would really like to get this list trimmed.